tests/deepdisc/utils/test_parse_arguments.py

```diff
@@ -11,7 +11,7 @@
 
 def test_make_inference_arg_parser():
     parser = make_inference_arg_parser()
-    args = parser.parse_args()
+    args = parser.parse_known_args()[0]
```

Comment:
I *think* you could test this by passing in the command line arguments to parse explicitly. This way, you could test different inputs without needing to run from the command line (and ignore any other conflicting arguments from the pytest invocation). Something like `parser.parse_args([])`.

Then if you wanted to test specific inputs you'd do something like `parser.parse_args(["--flag", "test_value"])`

Reply:
Changed to match your suggestion.